### ISSUE TITLE
[!FieldName] references not supported in new SharePoint Lists Experience

### ISSUE THREAD

**Issue by wwcoop** [SharePoint/sp-dev-docs#9854]
[!FieldName] references are not working in the new SharePoint Lists Experience.
Using this type of reference results in the JSON formatting not rendering.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 1aa6ae62-4dd8-3bdf-2f2c-f3f4a76e43b9
* Version Independent ID: 1aa6ae62-4dd8-3bdf-2f2c-f3f4a76e43b9
* Content: [Formatting syntax reference](https://learn.microsoft.com/en-us/sharepoint/dev/declarative-customization/formatting-syntax-reference)
* Content Source: [docs/declarative-customization/formatting-syntax-reference.md](https://github.com/SharePoint/sp-dev-docs/blob/main/docs/declarative-customization/formatting-syntax-reference.md)
* Service: **sharepoint-online**
* GitHub Login: @VesaJuvonen
* Microsoft Alias: **vesaj**

**Comment by Fedes365** [comment]
@wwcoop 

I've used **[!FieldName.DisplayName]**, both in a document library and in a list of the "New" Microsoft Lists as explained [here](https://learn.microsoft.com/sharepoint/dev/declarative-customization/formatting-syntax-reference#fieldname-1). Here is the result below with a very basic example. Does it work for you with this different reference?

![immagine](https://github.com/user-attachments/assets/c8c5834f-1012-4928-82c1-fdb62eb29f0e)

**Comment by Fedes365** [comment]
@wwcoop I'm not a member of sp-dev-docs staff so I cannot close this issue 😅. I think you can autonomously, there should be a button you can click to close it.

With regard to your JSON code of custom hovercards, I suggest that you open an issue [here](https://github.com/pnp/List-Formatting) so that somebody can have a deeper look at it and find a possible workaround.

See you,
Federico

**Comment by tpodugu-ms** [comment]
@wwcoop @Fedes365 I see the extra height being added to the custom card. Will work on this and update you the ETA.

**Comment by wwcoop** [comment]
Fantastic! Thanks @tpodugu-ms ! @Fedes365

**Comment by tpodugu-ms** [comment]
@wwcoop @Fedes365 Issue has been fixed and should be available in prod in 2 weeks.

**Comment by wwcoop** [comment]
@tpodugu-ms @Fedes365 Great! Thanks for the fix. Glad to see this solved! Cheers

**Comment by jgupta-msft** [comment]
@wwcoop : I hope this is fixed for you know.

**Comment by wwcoop** [comment]
@jgupta-msft Yes - it looks good. Thanks for solving this tricky issue. Thanks for your time and effort. Nice work!

**Comment by Amey-MSFT** [comment]
Hello @wwcoop,
Thank You for your patience. Is the issue resolved now?

**Comment by wwcoop** [comment]
Hi @Amey-MSFT - yes this issue is resolved. There is another JSON formatting issue that I reported in recently to support that does not have resolution:

- For a regular SharePoint list, go to list formatting, row formatting, advanced options.
- Apply any **border options** such as top and bottom borders for a row
- When you save out, no border options appear in the view

It looks like the border classes are applied in the HTML, but they are no longer rendered in the view. It seems this was a casualty of recent list UI updates. Border options did use to work, but somehow this broke as a result of updates. Other row formatting such as background colors still works, but not borders.

Can you investigate to see if this functionality can be restored?

**Comment by Amey-MSFT** [comment]
We are closing this issue for now. If the problem persists, feel free to reopen it or open a new one.

**Comment by wwcoop** [comment]
Hi @Amey-MSFT  - the original reported issue is resolved, but there is still this open issue I commented in thread:

- For a regular SharePoint list, go to list formatting, row formatting, advanced options.
- Apply any border options such as top and bottom borders for a row
- When you save out, no border options appear in the view

It looks like the border classes are applied in the HTML, but they are no longer rendered in the view. It seems this was a casualty of recent list UI updates. Border options did use to work, but somehow this broke as a result of updates. Other row formatting such as background colors still works, but not borders.

Should I open a separate request? Thanks very much for your time and help!
